tests/integration/remote/util.py

```diff
@@ -41,6 +43,23 @@ def remote_config_polygon(tmp_path):
     return config
 
 
+@enforce_types
+def get_gas_fees_for_remote() -> tuple:
+    # Polygon & Mumbai uses EIP-1559. So, dynamically determine priority fee
+    gas_resp = requests.get("https://gasstation-mainnet.matic.network/v2")
+
+    if not gas_resp or gas_resp.status_code != 200:
+        print(
+            f"Invalid response from Polygon gas station. Retry with brownie values..."
+        )
+
+        return chain.priority_fee, chain.base_fee + 2 * chain.priority_fee
+
+    return Web3.toWei(gas_resp.json()["fast"]["maxPriorityFee"], "gwei"), Web3.toWei(
```

Comment:
nice work. If I were to nitpick, I would recommend assigning gas_resp.json() e.g. `gas_resp = gas_resp.json()` before, to make this easier to read, but only if you want to. Also, there's an empty line in nonocean_tx after the try which I would remove, but that is also minor. I'm approving, feel free to merge now or with these tiny changes.